app/design/frontend/Magento/blank/web/css/source/_navigation.less

```diff
@@ -200,14 +201,12 @@
 
     .nav-open {
         .page-wrapper {
-            left: 80%;
             left: calc(~'100% - @{active-nav-indent}');
         }
 
         .nav-sections {
-            @_shadow: 0 0 5px 0 rgba(50, 50, 50, .75);
-
             .lib-css(box-shadow, @_shadow, 1);
+            @_shadow: 0 0 5px 0 rgba(50, 50, 50, .75);
```

Comment:
Could you explain why you moved variable declaration  below the using it?

Reply:
just follow magento standard sorting order should declare mixin first and so on following by other. Less still work without problem if you declare variable later. It's lazy evaluation. Variables do not have to be declared before being used.

Reply:
But here is not mixin declaration, but using mixin. Isn’t it?
As to me it’s not looks logical
Btw could you point me where such statement described

Reply:
Ok, now I see you moved variable declaration to the top, now it’s fine for me